Added constants generation.

theg4sh opened the pull request:
* Generation is skipped when MD5Sum is empty.
+ Added --make-copy flag to copy original msg/srv
  file into the same directory as output file.
  It makes correct pair of msg/srv and msg.go/srv.go files in
  output directory, e.g. if error is occured when parsing.
* Regerated msg files.

Commit:
Added constants generation.

* Generation is skipped when MD5Sum is empty.
+ Added --make-copy flag to copy original msg/srv
  file into the same directory as output file.
  It makes correct pair of msg/srv and msg.go/srv.go files in
  output directory, e.g. if error is occured when parsing.

angri reviewed (changes requested):
Constant names should be namespaced in some way. Otherwise constants with same names in a different messages of the same package would clash. Maybe add a prefix to names equal to message name, like `GoalStatus_PENDING = 0`. It doesn't look nice, but it's better than cashing names.

Commit:
Constants' prefixes

Prefixes of constants is the msg file name without .msg
That change required to prevent conflicts of similar constants'
names in different .msg files of one package
